Fitness/Views/Cells/HabitTrackerCheckinCell.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  HabitTrackerCheckinCell.swift
+//  Fitness
+//
+//  Created by Joseph Fulgieri on 4/23/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+
+class HabitTrackerCheckinCell: UICollectionViewCell {
+    
+    // MARK: - INITIALIZATION
+    static let identifier = Identifiers.habitTrackerCheckinCell
+    
+    private var checkinCircle: UIButton!
+    private var separator: UIView!
+    private var streakLabel: UILabel!
+    private var titleLabel: UILabel!
+    
+    private var habit: Habit!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        checkinCircle = UIButton()
+        checkinCircle.translatesAutoresizingMaskIntoConstraints = false
+        checkinCircle.setImage(UIImage(named: "empty_circle"), for: .normal)
+        checkinCircle.setImage(UIImage(named: "checked_circle"), for: .selected)
+        checkinCircle.addTarget(self, action: #selector(checkIn), for: .touchUpInside)
+        contentView.addSubview(checkinCircle)
+        
+        titleLabel = UILabel()
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.textColor = .fitnessBlack
+        titleLabel.font = ._16MontserratMedium
+        contentView.addSubview(titleLabel)
+        
+        streakLabel = UILabel()
+        streakLabel.translatesAutoresizingMaskIntoConstraints = false
+        streakLabel.textColor = .black
+        streakLabel.font = ._16MontserratBold
+        contentView.addSubview(streakLabel)
+        
+        separator = UIView()
+        separator.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
you don't need to set `.translatesAutoresizing...` when you use SnapKit so should remove